app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -50,6 +51,16 @@ sealed class Wallpaper {
         data class Focus(override val name: String, override val expirationDate: Date? = null) : Remote() {
             override val remoteParentDirName: String = "focus"
         }
```

Comment:
Should we possibly remove this from nightly as well, now that there will be a more official remote wallpaper available?

Reply:
Yes, we can remove it :)